src/group_theory/submonoid/membership.lean

```diff
@@ -233,12 +233,19 @@ lemma pow_right_injective_iff_pow_injective {n : M} :
   function.injective (λ m : ℕ, n ^ m) ↔ function.injective (pow n) :=
 subtype.coe_injective.of_comp_iff (pow n)
 
-theorem log_pow_eq_self [decidable_eq M] {n : M} (h : function.injective (λ m : ℕ, n ^ m)) (m : ℕ) :
```

Comment:
Is this lemma being deleted? I don't see it moving elsewhere

Reply:
This is now just the `left_inv` in the equivalence. See, for example, in the proof of `log_pow_int_eq_self` below.

Reply:
On the other hand, `pow_log_eq_self` cannot be derived from the equivalence as it does not depend on the injectivity of `pow`.

Reply:
@apurvnakade I agree, but I think that for discoverability it still makes sense to keep the lemma.
Also, the `left_inv` will not be syntactically equal to this lemma. So I would prefer to keep it.

Reply:
I put it back in. Should this theorem also have a `[simp]` tag? It did not have one originally.

Reply:
i'd think so